apps/notifications/index.jsx

```diff
@@ -279,10 +274,11 @@ export class Notifications extends Component {
 export default connect(
 	( state ) => ( {
 		currentLocaleSlug: getCurrentLocaleVariant( state ) || getCurrentLocaleSlug( state ),
+		forceRefresh: shouldForceRefresh( state ),
```

Comment:
Update: Confirmed that the Redux event is firing in Calypso, but is not being picked up here (i.e. doesn't trigger a re-render of the component). I'm not quite sure how the existing `getCurrentLocaleVariant` getter works in this case.

<p align="center">
<img width="300" alt="notifications-refresh-calypso" src="https://user-images.githubusercontent.com/8979548/94833434-1df23380-03dd-11eb-9881-4e29aeace3f4.png">
</p>

Reply:
Update: finally got this working thanks to corrections proposed by @tyxla (d7aec3a). Works like a charm now - thank you for your help!

Reply:
Glad it worked, @nsakaimbo 🎉